lib/elements/tab_expand_controller.dart

```diff
@@ -175,6 +181,7 @@ class TabExpandController {
     consoleButton.toggleClass('active', false);
     docsButton.toggleClass('active', false);
     closeButton.toggleAttr('hidden', true);
+    clearConsoleButton.setAttr('style', 'visibility:hidden;');
```

Comment:
Why not `toggleAttr('hidden', true);`?

Reply:
I needed the element to be hidden but still measurable - (so the close button doesn't move when it's hidden).
I don't know if I tried using the hidden element attribute, I will give that a try... (and I will fix that if that works too)

Reply:
Yes we must use the style here, because toggling the hidden attribute removes the element from the size calculations for the page and the close button moves.  Once I went to the code I remembered trying the toggleAttr('hidden'....) first for consistency reasons, but it didn't work because the close button would move when the documentation tab was active.

Reply:
i could add an explicit comment to the code explaining that..

Reply:
that's fine, thanks for checking that.